src/applications/vaos/components/PastAppointmentsList.jsx

```diff
@@ -10,24 +10,22 @@ import { getPastAppointmentDateRangeOptions } from '../utils/appointment';
 import ConfirmedAppointmentListItem from './ConfirmedAppointmentListItem';
 import PastAppointmentsDateDropdown from './PastAppointmentsDateDropdown';
 
-const dateRangeOptions = getPastAppointmentDateRangeOptions();
-
 export class PastAppointmentsList extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
-      selectedDateRange:
-        dateRangeOptions[this.props.appointments.pastSelectedIndex],
-    };
+    this.dateRangeOptions =
+      props.dateRangeOptions || getPastAppointmentDateRangeOptions();
```

Comment:
Do we ever expect to get the `dateRangeOptions` via props?  

On another note, I probably should have just put `getPastAppointmentDateRangeOptions()` in this file instead of `appointment.js`

Reply:
It's only used in tests. We could probably remove it. It's being really cautious about making sure that the exact same date range, relative to a past date instead of today, is used in both the test and the component. But calling that get function twice within a couple seconds (at worst) is unlikely to cause a mismatch that fails a test.

Reply:
Actually, it's probably easier to leave it. I thought the options list was just the date, not the time, too.